app/models/journal/aggregated_journal.rb

```diff
@@ -41,6 +41,12 @@
 #    be dropped
 class Journal::AggregatedJournal
   class << self
+    # Returns the aggregated journal that contains the specified (vanilla) journal.
+    def for_journal(vanilla_journal)
+      Journal::AggregatedJournal.aggregated_journals(journable: vanilla_journal.journable)
+                  .detect { |journal| journal.version >= vanilla_journal.version }
```

Comment:
true.
and now that my other PR is merged, there is already a helper method for that SQL.... will do that!